src/components/Modeler.vue

```diff
@@ -42,29 +42,42 @@
           <span class="btn btn-sm btn-secondary scale-value">{{ Math.round(scale*100) }}%</span>
         </div>
 
-        <button class="btn btn-sm btn-secondary mini-map-btn ml-auto" data-test="mini-map-btn" @click="miniMapOpen = !miniMapOpen">
-          <font-awesome-icon v-if="miniMapOpen" :icon="minusIcon" />
-          <font-awesome-icon v-else :icon="mapIcon" />
-        </button>
+        <div class="btn-group btn-group-sm mr-2" role="group" aria-label="Third group">
```

Comment:
Nitpick: Instead of having the aria labels be "First group", "Second group" and "Third group", I think more meaningful names like "Undo/redo controls", "Zoom controls" and "Additional controls" would be better.